### DISCUSSION TITLE
Was geopackage worth it?

### DISCUSSION THREAD

**Discussion by philipbaileynar** [Riverscapes/riverscapes-tools#128] [General] [answered]
We put a lot of effort in. Might still be some bugs. Was it worth the additional effort?

**Comment by MattReimer** [comment, marked as answer]
I'd say yes

**Reply by MattReimer** [reply]
Thanks to #170  and #116 I'd say yes.

**Reply by philipbaileynar** [reply]
Funny, I am actively working on lingering trickle down effects in RVD. It was worth it, but it requires all developers to be moderately DB savvy to both do it right and also to leverage the full power of the database.

**Reply by MattReimer** [reply]
Yeah. I'm more wondering how to make discussions useful. This one doesn't really have a purpose or any good threads on it. Do we delete it or click the checkmark to answer it?

**Reply by MattReimer** [reply]
Although your latest comment makes this thread kind of useful. It's tricky. There's no "close" button on discussions so they just hand around..... hmmmm

**Reply by philipbaileynar** [reply]
I actually think that's a good thing for end user questions. Always searchable. Never closed. Can be hard for newbs to find closed issues.

**Reply by joewheaton** [reply]
Absolutely!
